onnx/onnxifi_ext.h

```diff
@@ -0,0 +1,163 @@
+ *                                          ModelProto.graph.output doesn't have
+ *                                          a match in the inputDescriptors or
+ *                                          outputDescriptors.
+ * @retval ONNXIFI_STATUS_MISMATCHING_SHAPE The function call failed because
+ *                                          the shapes specified through
+ *                                          inputDescriptors or
+ *                                          outputDescriptors argument are
+ *                                          inconsistent with the shapes
+ *                                          specified in the ONNX model graph.
+ * @retval ONNXIFI_STATUS_MISMATCHING_DATATYPE The function call failed because
+ *                                             data types specified through
+ *                                             inputDescriptors or
+ *                                             outputDescriptors argument are
+ *                                             inconsistent with the data types
+ *                                             specified in the ONNX model
+ *                                             graph.
+ * @retval ONNXIFI_STATUS_NO_SYSTEM_MEMORY The function call failed because the
+ *                                         backend could not allocate enough
+ *                                         system memory to parse, analyze, and
+ *                                         initialize the tensor locations.
+ * @retval ONNXIFI_STATUS_NO_SYSTEM_RESOURCES The function call failed due to
+ *                                            insufficient non-memory system
+ *                                            resources (e.g. file handles) to
+ *                                            initialize the tensor locations.
+ * @retval ONNXIFI_STATUS_NO_DEVICE_MEMORY The function call failed due to
+ *                                         insufficient backend-specific memory
+ *                                         to initialize the tensor locations.
+ * @retval ONNXIFI_STATUS_NO_DEVICE_RESOURCES The function call failed due to
+ *                                            insufficient non-memory
+ *                                            backend-specific resources (e.g.
+ *                                            command queues) to initialize the
+ *                                            tensor locations.
+ * @retval ONNXIFI_STATUS_BACKEND_UNAVAILABLE The function call failed because
+ *                                            the backend was disconnected or
+ *                                            uninstalled from the system.
+ * @retval ONNXIFI_STATUS_INTERNAL_ERROR The function call failed because the
+ *                                       backend experienced an unrecovered
+ *                                       internal error.
+ */
+
+ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI
```

Comment:
You need a typedef to define a new function type. Check https://github.com/onnx/onnx/pull/1757/files#diff-8312e1aaff6ed2027647302d102f0d44R271 for an example.